src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -489,6 +489,31 @@ typedef struct {
     DOMString decoderImplementation; //!<  TODO Identifies the decoder implementation used. This is useful for diagnosing interoperability issues.
 } RtcInboundRtpStreamStats, *PRtcInboundRtpStreamStats;
 
+/**
+ * Reference: https://www.w3.org/TR/webrtc/#dom-rtcdatachannelstate
+ */
+typedef enum {
+    RTC_DATA_CHANNEL_STATE_CONNECTING,
+    RTC_DATA_CHANNEL_STATE_OPEN,
+    RTC_DATA_CHANNEL_STATE_CLOSING,
+    RTC_DATA_CHANNEL_STATE_CLOSED
+} RTC_DATA_CHANNEL_STATE;
+
+/**
+ * Reference: https://www.w3.org/TR/webrtc-stats/#dom-rtcdatachannelstats
+ */
+struct {
```

Comment:
Sorry. Missed this earlier. But, this needs to typedef'd as well. That is probably why the CI is failing with link errors.

Reply:
its embarrassing :-/